draft-ietf-tls-certificate-compression.md

```diff
@@ -85,27 +91,28 @@ CertificateCompressionAlgorithms value:
     } CertificateCompressionAlgorithms;
 ~~~
 
-If the server supports any of the algorithms offered in the ClientHello, it MAY
-respond with an extension indicating which compression algorithm it chose.  In
-that case, the "extension_data" SHALL be a CertificateCompressionAlgorithm value
-corresponding to the chosen algorithm.  If the server has chosen to not use any
-compression, it MUST NOT send the compress_certificates extension.
+There is no ServerHello extension that the server is required to echo back.
```

Comment:
Smallest of nitpicks: is this sentence necessary? It's already not exclusively a client extension since the spec says it is in ClientHello or CertificateRequest. (Also TLS 1.3 typically acknowledges extensions in EncryptedExtensions rather than ServerHello anyway.)

Reply:
It's not necessary, however, I do feel like it's helpful to call out explicitly.